src/config/services/index.md

```diff
@@ -32,3 +37,136 @@ Optionally, a service directory may contain:
 
 When a new service is created, a `supervise` folder will be automatically
 created on the first run.
+
+### Configuring Services
+
+Most services can take configuration options set by a `conf` file in the service
+directory. This allows service customization without modifying the service
+directory provided by the relevant package.
+
+Check the service file for how to pass configuration parameters. A few services
+have a field like `OPTS="--value ..."` in their `conf` file.
+
+To make more complex customizations, you should [edit the
+service](#editing-services).
+
+### Editing Services
+
+To edit a service, first copy its service directory to a different directory
+name. Otherwise, [xbps-install(1)](https://man.voidlinux.org/xbps-install.1) can
+overwrite the service directory. Then, edit the new service file as needed.
+Finally, the old service should be stopped and disabled, and the new one should
+be started.
+
+## Managing Services
+
+### Runsvdirs
+
+A `runsvdir` is a directory in `/etc/runit/runsvdir` containing enabled services
```

Comment:
Upon further thought, i think we shouldn't use backticks unless we're referring to a command. In this instance, we're referring to a directory, not a command, so perhaps we should use "\*runsvdir\* instead, using emphasis because we're defining a term.

Reply:
I think I will only leave the first one bolded. Sounds good?

Reply:
Sure. :-)